lib/screens/animal_inventory_screen.dart

```diff
@@ -131,14 +138,14 @@ Widget inventoryListTile(BuildContext context, Animal animal, String userType) {
               children: <Widget>[
                 Expanded(
                   flex: 3,
-                  child: CircleAvatar(
-                    radius: 100.0,
-                    backgroundColor: Colors.transparent,
-                    backgroundImage: NetworkImage(animal.imageURL),
+                  child: ClipOval(
```

Comment:
Does this make it look oval now? I tried the ClipOval when I was redesigning, and I didn't like how the images were not the same shape for landscape and portrait animal images. That's why I went with the CircleAvatar.